stl.go

```diff
@@ -209,20 +209,32 @@ func ReadFromSTL(i io.Reader) (o *Subtitles, err error) {
 
 		// Parse TTI block
 		var t = parseTTIBlock(b, g.framerate)
-
+		
 		if t.extensionBlockNumber != extensionBlockNumberReservedUserData {
 
+			var starttime=g.timecodeStartOfProgramme
+			if starttime>t.timecodeIn{
+				starttime=0
+			}
 			// Create item
 			var i = &Item{
-				EndAt:   t.timecodeOut - g.timecodeStartOfProgramme,
-				StartAt: t.timecodeIn - g.timecodeStartOfProgramme,
+				EndAt:   t.timecodeOut - starttime,
+				StartAt: t.timecodeIn - starttime,
 			}
 
 			// Loop through rows
 			for _, text := range bytes.Split(t.text, []byte{0x8a}) {
-				parseTeletextRow(i, ch, func() styler { return newSTLStyler() }, text)
+				parseTeletextRow(i, ch, func() styler { return newSTLStyler() }, text)				
 			}
 
+			if i.InlineStyle!=nil{
+				fmt.Printf("InlineStyle not NULL\n")
+			}else{
+				i.InlineStyle=&StyleAttributes{}
+			}
+			i.InlineStyle.TeletextVerticalPostion=t.verticalPosition
+			i.InlineStyle.TeletextJustificationCode=t.justificationCode
+			i.InlineStyle.propagateTeletextPositionJustification()
```

Comment:
I'd rather use the `propagateSTLAttributes` function, that exists but is empty. Therefore, you'd have to move the logic you've created into this function.